sdcm/cluster.py

```diff
@@ -5122,6 +5122,8 @@ def node_setup(self, node, verbose=False, **kwargs):  # pylint: disable=unused-a
         # Install java-8 for workaround hdrhistogram
         if node.distro.is_rhel_like:
             node.install_package(package_name='java-1.8.0-openjdk-devel')
+            node.install_package(package_name="libc6-compat")
+            node.install_package(package_name="gcompat")
```

Comment:
Don't we use docker based loaders?
In that case, maybe it's missing from the docker image?
But what has changed?

Reply:
It's the new c-s images, and seems we are missing snappy

We should raise it and get Dusan to fix it

Reply:
@CodeLieutenant do you have an issue for it?
If not, please create yourself one.

Reply:
@CodeLieutenant please share the link to the issue.

Reply:
@CodeLieutenant I suppose this is it https://github.com/scylladb/cassandra-stress/issues/13?

Reply:
> @CodeLieutenant I suppose this is it [scylladb/cassandra-stress#13](https://github.com/scylladb/cassandra-stress/issues/13)?

Yes it was, the issue is fixed in the latest c-s release 3.13.0, I'll be making a pull request to SCT as soon as I'm done testing it.